src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -486,8 +486,15 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 		this.layout(this.dimensions);
 	}
 
-	openEditor(editor: EditorInput): boolean {
-		return this.handleOpenedEditors();
+	openEditor(editor: EditorInput, options?: IInternalEditorOpenOptions): boolean {
+		const changed = this.handleOpenedEditors();
```

Comment:
@benibenj I think the `focus()` call needs to happen after we opened the editor because the new editor might be in a new tab that is created. So I moved it down.

Reply:
I put it first as it might feel a bit more responsive if it is called earlier. With the current state this is not a problem, but for future changes which might use this option, your solution is safer.